assignment2.yaml

```diff
@@ -0,0 +1,138 @@
+swagger: "2.0"
+info:
+  description: "Welcome to the swapi, the Star Wars API! This documentation should help you familiarise yourself with the resources available and how to consume them with HTTP requests. If you're after a native helper library then I suggest you scroll down and check out what's available. Read through the getting started section before you dive in. Most of your problems should be solved just by reading through it."
+  version: ""
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/documentation"
+tags:
+- name: "Films"
+  description: "A Film resource is a single film."
+schemes:
+- "http"
+paths:
+  /films/:
+    get:
+      tags:
+      - "Films"
+      summary: "get all the film resources"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: ":id"
+        in: "query"
+        description: "id of the film to be returned"
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "OK"
+          schema:
+            $ref: "#/definitions/Films"
+  /films/{:id}/:
+    get:
+      tags:
+      - "Films"
+      summary: "get a specific film resource"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: ":id"
+        in: "path"
+        description: "id of the film to be returned"
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "OK"
+          schema:
+            $ref: "#/definitions/Films"
+  /films/schema/:
+    get:
+      tags:
+      - "Films"
+      summary: "view the JSON schema for this resource"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "OK"
+          schema:
+            $ref: "#/definitions/Films"
```

Comment:
Schema API doesn't return you with this response! This response is returned by _films/:id_ API. First hit the API and then document its response!